src/Runner.js

```diff
@@ -127,7 +127,7 @@ function run(transformFile, files, options) {
   file_chunks.forEach(function(files) {
     var child = child_process.fork(
       require.resolve('./Worker'),
-      [transformFile]
+      [transformFile, options.babel ? 'babel' : 'no-babel']
```

Comment:
I could use `true` and `false` or `1` and `0` or any other string but I think this is the most explicit and easiest to understand.